Build image for Pi Zero 2 W

ardangelo opened the pull request:
Also:

* Optimize boot services
* Resize home partition to fill SD card
* Copy WLAN and timezone configuration from boot partition
* Sync clock to RTC
* Add mosh, aerc, w3m, Python

ardangelo committed:
Buildroot included

ardangelo committed:
Update driver branches to master

ardangelo committed:
Optimize boot services

ardangelo committed:
Move init scripts at build time

ardangelo committed:
Format and resize home partition on first boot

ardangelo committed:
Set timezone, add mosh, python

ardangelo committed:
Sync hardware clock on boot, fix first boot partition resize

ardangelo committed:
Add w3m, notmuch, aerc

ardangelo committed:
Add connection status indicator

ardangelo committed:
Copy WLAN PSK files from /boot/wlan at startup

ardangelo committed:
Update keyboard and display module names

ardangelo committed:
Update keyboard and display module names

ardangelo committed:
Implement shutdown and startup using power key (requires firmware v2.1)

ardangelo committed:
Update keyboard driver to main branch

ardangelo committed:
Add Buildroot workflow

ardangelo committed:
Checkout submodules in workflow

ardangelo committed:
Add staging directories to notmuch flags

ardangelo committed:
Move Buildroot scripts out into repo

ardangelo committed:
Add build for Zero 2 W

ardangelo committed:
Add updater script

ardangelo committed:
Sync whole root partition

ardangelo committed:
Add nmcli

ardangelo committed:
Exclude more directories from update

maxmbed commented:
In change 1e89e17f521f332eeb1253ef383885b9941c6ef0, you are superseding the rpi0 board by rpi0-2w without clear notice.
Maybe defconfig, genimage and any board specific config files can can be moved into a board directory ? 

Such as:
```bash
./board/
├── br_defconfig-rpi0
├── br_defconfig-rpi0-2w
├── linux_defconfig-rpi0
└── linux_defconfig_rpi0-2w
```
By the way, naming kernel and buildroot config accordingly with a prefix would help to differentiate.

ardangelo committed:
Update beepy-kbd to 2.3 and sharp-drm to 1.1

Commit:
Update post-build.sh to silence boot error related to missing sgx group

Something in udev would really prefer it if there was an "sgx" group in existence at boot time.

Commit:
Missing colon

ardangelo committed:
Update post-build.sh to silence boot error related to missing sgx group

ardangelo committed:
Add symbol menu overlay

ardangelo committed:
Update drivers, fix Unicode symbol keys

ardangelo committed:
Add tmux-menus script

ardangelo committed:
Add pip to Python3

ardangelo committed:
Update readme

ardangelo committed:
Update gomuks and add libolm

ardangelo committed:
Update Buildroot to 2024.02, beepy-kbd to 2.9, libgc to 8.2.6, fix aerc build

ardangelo committed:
Use public golang proxy

ardangelo committed:
Create LICENSE

ardangelo committed:
Add GCC, SDL2, other deveolpment utilities

jdagdelen committed:
Specify build script is bash only

Changing shebang to bash because build script requires pushd

ardangelo committed:
Merge pull request #7 from jdagdelen/master

Specify build script is bash only
